app/services/feature_flag.rb

```diff
@@ -65,12 +68,30 @@ def self.deactivate(feature_name)
   def self.active?(feature_name)
     raise unless feature_name.in?(FEATURES)
 
-    FEATURES[feature_name].feature.active?
+    feature_statuses[feature_name].presence || false
   end
 
   def self.sync_with_database(feature_name, active)
     feature = Feature.find_or_initialize_by(name: feature_name)
     feature.active = active
+
+    clear_cache!
```

Comment:
should we clear the cache after saving? just in case the cache happened to be renewed by an incoming web request between the clearing and the saving, which would mean the change wouldn't show up in the cache